cmd/aws_signing_helper/main.go

```diff
@@ -80,18 +85,18 @@ func findGlobalVar(argList []string) (map[string]string, []string) {
 // Assigns different flags to different commands
 func setupFlags() {
 	for command, fs := range commands {
-		// applicable to `sign-string` and `credential-process` operation
-		if command == "sign-string" || command == "credential-process" {
+		// applicable to `sign-string`, `credential-process`, and `update` operations
+		if command != "read-certificate-data" {
```

Comment:
Feels like this will cause problems down the road (e.g., a new command will get implicitly included). Use an explicit allow list instead.

Reply:
Addressed in the next revision.